clients/venice-samza/src/main/java/com/linkedin/venice/samza/VeniceSystemProducer.java

```diff
@@ -744,6 +770,38 @@ private static <T> byte[] serializePrimitive(T input, DatumWriter<T> writer) {
     return out.toByteArray();
   }
 
+  private Object convertWriteComputeRecordToFullPut(
```

Comment:
nit: Not necessary to fix here but we plan to formalize the naming to partial update everywhere. It could happen in the future renaming PR though.

Reply:
Good call out, I'll tweak the name.